src/library/scala/collection/immutable/Stream.scala

```diff
@@ -451,7 +447,7 @@ object Stream extends SeqFactory[Stream] {
     */
   def iterate[A](start: => A)(f: A => A): Stream[A] = {
     lazy val head = start
-    newCons(head, iterate(f(head))(f))
+    Stream.cons(head, iterate(f(head))(f))
```

Comment:
Not a huge deal, but I don't think the `newCons` replacements need to be prefixed with `Stream.`